packages/ember-views/lib/views/states/has_element.js

```diff
@@ -15,8 +16,21 @@ var hasElement = Object.create(_default);
 
 assign(hasElement, {
   $(view, sel) {
-    var elem = view.element;
-    return sel ? jQuery(sel, elem) : jQuery(elem);
+    if (isEnabled('ember-views-tagless-jquery') && view.tagName === '') {
+      var firstNode = view._renderNode.firstNode;
+      var lastNode = view._renderNode.lastNode;
+      var range = jQuery(firstNode).nextUntil(lastNode);
+
+      //Test if we should include first and last nodes in the range.
+      if (firstNode.nodeType !== 8) { range = range.add(firstNode); }
+      if (lastNode.nodeType !== 8) { range = range.add(lastNode); }
+
+      //Return inner find and topmost elements filter if a selector is provided
+      return sel ? jQuery(sel, range).add(range.filter(sel)) : range;
```

Comment:
can you explain this? Doesn't 'jQuery(sel, range)' do the work we need? I guess what I'm asking is, what is the result of the '.add(range.filter(sel))' bit?